### PR TITLE
Disable drag & drop in tables [ESD-1282]

### PR DESCRIPTION
**Pull request by tn-exafore**
* also disable editing in some tables where it was previously implicitly allowed but makes no sense

### CODE REVIEW

File: piksi_tools/console/system_monitor_view.py [hunk 1]
```diff
@@ -32,10 +32,18 @@ def ip_bytes_to_string(ip_bytes):
...
     columns = [('Thread Name', 0), ('CPU %', 1), ('Stack Free', 2)]
+    can_edit = Bool(False)
+
+    def get_drag(self, object, trait, row):
+        return None  # disables drag
 
 
 class SimpleCSACAdapter(TabularAdapter):
     columns = [('Metric Name', 0), ('Value', 1)]
+    can_edit = Bool(False)
```
Comment (benjaminaltieri) [inline piksi_tools/console/system_monitor_view.py]: this looks like shotgun surgery, can class level variables be inherited? maybe a superclass that all of these SimpleAdapters can inherit would reduce duplicated logic here.

Comment (tn-exafore) [reply piksi_tools/console/system_monitor_view.py]: Agreed, it was a bit rushed. Refactored.

**Comment by tn-exafore** [comment]
I'll "automatically" backport this to release-v2.3.0 when this PR is approved if nobody objects to it.

File: piksi_tools/console/output_list.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
 from traits.api import (Bool, Enum, Float, Font, HasTraits, Int, List,
                         Property, Str, Trait)
 from traitsui.api import TabularEditor, UItem, View
-from traitsui.tabular_adapter import TabularAdapter
+from .gui_utils import PiksiTabularAdapter
```
Comment (benjaminaltieri) [inline piksi_tools/console/output_list.py]: full path import

File: piksi_tools/console/solution_view.py [hunk 3]
```diff
@@ -34,7 +34,7 @@
                         Str, Enum)
 from traitsui.api import (HGroup, HSplit, Item, TabularEditor, TextEditor,
                           VGroup, View)
-from traitsui.tabular_adapter import TabularAdapter
+from .gui_utils import PiksiTabularAdapter
```
Comment (benjaminaltieri) [inline piksi_tools/console/solution_view.py]: Same here

File: piksi_tools/console/gui_utils.py [hunk 4]
```diff
@@ -130,3 +131,10 @@ def get_filter_group():
                         visible_when="{} in received_codes".format(code)))
             hgroup.content.append(vgroup)
         return hgroup
+
+
+class PiksiTabularAdapter(TabularAdapter):
```
Comment (benjaminaltieri) [inline piksi_tools/console/gui_utils.py]: this looks great, piksi is a bit of a misnomer though, maybe `NoEditTabularAdapter`? Or something version that denotes what is being inherited

Comment (tn-exafore) [reply piksi_tools/console/gui_utils.py]: There is some chance of further style etc. updates in the future. Anyway, renamed to `ReadOnlyTabularAdapter`, it is a descriptive name at the moment.

File: piksi_tools/console/output_list.py (hunk 2 above)
Comment (tn-exafore) [reply piksi_tools/console/output_list.py]: fixed

**Pull request merged**